### PR TITLE
Creates new tab for barnstars

### PR DESCRIPTION
**Pull request by carolineh101**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] All tests pass -- `rake test:all`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request are descriptively named
* [x] if possible, multiple commits squashed if they're smaller changes
* [x] reviewed/confirmed/tested by another contributor or maintainer
* [x] `schema.rb.example` has been updated if any database migrations were added

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

File: app/views/users/profile.html.erb [hunk 1]
```diff
@@ -195,6 +189,15 @@
     <div class="tab-pane" id="comments">
       <h2 style="text-align:center;"><i class="fa fa-spinner fa fa-spin"></i></h2>
     </div>
+
+    <div class="tab-pane" id="barnstars">
+      <% if @user.user %>
+      <% @user.user.barnstars.each do |tag| %>
+        <p style="color:#aaa;"><i style="color:#db4;" class="fa fa-large fa fa-star-o"></i> <i><%= raw t('users.profile.barnstar_awarded_to_by', :url1 => "/profile/"+tag.node.author.name, :author => tag.node.author.name, :url2 => "/wiki/barnstars#"+tag.name.split('-').each{|w| w.capitalize!}.join('+'), :barnstar => tag.name.split(':').last.split('-').each{|w| w.capitalize!}.join(' '), :url3 => "/profile/"+tag.author.username, :awarder => tag.author.username, :url4 => tag.node.path, :work => tag.node.title) %></i></p>
```
Comment (ananyo2012) [inline app/views/users/profile.html.erb]: There is a little bug out there that displays the incorrect url. `:url2` will be `"/wiki/barnstars#"+tag.name.split(':').last.split('-').each{|w| w.capitalize!}.join('+')`

**Comment by jywarren** [comment]
Hi! Hope to get to this during tomorrow's merge, thanks! Worst case, Wednesday. It looks great!

**Comment by ananyo2012** [comment]
Well I see some earlier commits which has been already merged in the master. I think this happened as there wasn't a clean fast-forward merge.

**Comment by jywarren** [comment]
Oh, hmm, well I'll rebase it over the latest anyways, so it should be OK. I reviewed the code and it looks good!

**Comment by jywarren** [comment]
Rebasing & testing...

all good! Thanks! There are a [bunch of new help-wanted issues](https://github.com/publiclab/plots2/labels/help-wanted) that've been posted recently, and I'd also be happy to think through any that'd be especially interesting to you. Is there a part of the site that you'd like to get involved in reworking? There are a [lot of possibly interesting ones here](https://github.com/publiclab/plots2/labels/outreach) although some may be out of date.

**Pull request closed without merging**